### PR TITLE
Repair docker example. Solves #9476

### PR DESCRIPTION
**Pull request by AliyeSimsek**

### CODE REVIEW

**Comment by PRMerger7** [comment]
@AliyeSimsek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by lindt** [comment]
@ktoliver : Kristine, any updates on this?

**Comment by ktoliver** [comment]
> @ktoliver Kristine Toliver (Aquent LLC) Vendor : Kristine, any updates on this?

Hi @lindt - An automated review request was sent to the author. 

@ramiMSFT - Could you review the proposed update to your article? Thanks!

File: docs/pipelines/release/caching.md [hunk 1]
```diff
@@ -454,7 +454,7 @@ pool:
 steps:
   - task: Cache@2
     inputs:
-      key: 'docker | "$(Agent.OS)" | caching-docker.yml'
+      key: 'docker | $(Agent.OS)'
```
Comment (ramiMSFT) [inline docs/pipelines/release/caching.md:457]: revert to 'docker | "$(Agent.OS)" | caching-docker.yml'

**Review by ramiMSFT — CHANGES_REQUESTED** [review]
'docker | "$(Agent.OS)" | caching-docker.yml'

**Comment by ktoliver** [comment]
@AliyeSimsek - Hi, could you make the requested changes in your pull request? Thanks!

**Comment by PRMerger14** [comment]
@AliyeSimsek : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/release/caching.md (hunk 1 above)
Comment (AliyeSimsek) [reply docs/pipelines/release/caching.md]: Done :)

**Review by ramiMSFT — APPROVED** [review]

**Comment by lindt** [comment]
- solves #9044 
- solves #6280

thx @ktoliver , @ramiMSFT

**Pull request merged**